source/_integrations/reolink.markdown

```diff
@@ -54,9 +54,16 @@ Depending on the supported features of the camera, binary sensors are added for:
 - AI pet detection
 - AI face detection
 
-These sensors are polled every 60 seconds and receive ONVIF push events for immediate updates.
+These sensors receive events using 3 methods in order: ONVIF push, ONVIF long polling or fast polling (every 5 seconds).
+The latency for receiving the events is the best for ONVIF push and the worst for fast polling, the fastest available method that is detected to work will be used, and slower methods will not be used. The `Event Connection` sensor shows the current method being used.
+As a extra redundancy these sensors are polled every 60 seconds together with the update of all other entities.
 Not all camera models generate ONVIF push events for all event types, some binary sensors might, therefore, only be polled.
 For list of Reolink products that support ONVIF see the [Reolink Support Site](https://support.reolink.com/hc/en-us/articles/900000617826).
+If the `Event Connection` sensor does not show `ONVIF push` and you wish to reduce the latency, see the troubleshooting section.
```

Comment:
```suggestion
If the `Event Connection` sensor does not show `ONVIF push` and you wish to reduce the latency, refer to the [troubleshooting](#troubleshooting) section.
```
Is there a section on latency in the troubleshooting section? To me, it reads more like general network troubleshooting. Would it make sense to maybe explicitly mention the effect on latency in the troubleshooting step itself?

Reply:
I have removed this sentence now that the Event Connection sensor has been removed.
Will add something about latency in a seperate docs PR.

Reply:
I added new documentation about this here: https://github.com/home-assistant/home-assistant.io/pull/28288